src/library.coffee

```diff
@@ -80,8 +82,10 @@ componentsFromDirectory = (directory, config, callback) ->
 
 normalizeConfig = (config) ->
   config = {} if not config
+  namespace = config.name or null
```

Comment:
@jonnor this is to default namespace to NPM package name, like we do with NoFlo.